.gitpod.Dockerfile

```diff
@@ -1,7 +1,18 @@
 FROM gitpod/workspace-full
 
+
+FROM gitpod/workspace-full
```

Comment:
This is the same as line 1. Is it meant to be duplicated?

Reply:
ah no, it's totally a mistake (bad copy and past from Perses 😅. I will remove it.